hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/RawLocalFileSystem.java

```diff
@@ -293,15 +294,12 @@ public FileDescriptor getFileDescriptor() throws IOException {
 
     @Override
     public boolean hasCapability(String capability) {
-      // a bit inefficient, but intended to make it easier to add
-      // new capabilities.
       switch (capability.toLowerCase(Locale.ENGLISH)) {
       case StreamCapabilities.IOSTATISTICS:
       case StreamCapabilities.IOSTATISTICS_CONTEXT:
-      case StreamCapabilities.VECTOREDIO:
         return true;
       default:
-        return false;
+        return hasVectorIOCapability(capability);
```

Comment:
Does this actually change the logic? It seems like `hasVectorIOCapability` checks `StreamCapabilities.VECTOREDIO`, just like the old code did.